frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -66,7 +66,7 @@ function GameStatistics({ finalScores, onNewGameRequested }) {
         <div className="table-item">{getManualSubstractedGoals(TeamID.Team_blue)}</div>
         <div className="table-item">Manually substracted goals</div>
         <div className="table-item">{getManualSubstractedGoals(TeamID.Team_white)}</div>
-        <NumberOfShots statistics={statistics} />
+        {statistics && <NumberOfShots statistics={statistics} />}
```

Comment:
Now you aren't displaying `NumberOfShots` when statistics aren't provided, maybe better option is to add
```
if(statistics) return <p>Something go wrong, statistics goes on the vacation and we don't have it</p>
```
Just an option and sugestion.

Reply:
Thanks for that, we used it.